src/utils/permalinks/MatrixSchemePermalinkConstructor.ts

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
```

Comment:
Assuming this is new code, should be a new date?

```suggestion
Copyright 2022 The Matrix.org Foundation C.I.C.
```